### DISCUSSION TITLE
Illustrations - integrating into console

### DISCUSSION THREAD

**Discussion by jweintraub** [twilio-labs/paste#1410] [Ideas] [answered]
I had a question come up from a product team about how to serve an illustration in the Console with a link to rather than having to download it from Bynder and then add it manually. It is ideal to use the bynder repository as our source of truth for illustration assets so that we're using shared assets and any changes or updates to an illustration are automatically done everywhere rather than having to be manually updated in one or more code bases. I spoke with Sang Seo from the brand team who shared the following information:

"You can retrieve the assets via Bynder API. By default, the URLs to view the derivatives of image assets will be returned by the API. When you visit this type of URL the derivative will be displayed in the browser. If you marked the assets as public the link to view the original image file will also be returned. These ‘viewing’ links are valid permanently. You can find more information about this type of API call [here](https://urldefense.com/v3/__https://bynder.docs.apiary.io/*reference/assets/asset-operations/retrieve-assets__;Iw!!NCc8flgU!IRZFchUaga5BPvByXDA1IB0E1los33acXQBmgao7_TCiqS6uDIK_2kSE0iQ$)."

It would be great if we could add these instructions into the Paste illustration area. LMK your thoughts.

**Comment by nkrantz** [comment]
Hi @jweintraub - just want to let you know that we're discussing this internally and will respond soon with some options to move forward. Stay tuned!

**Reply by jweintraub** [reply]
Thank you!

**Comment by TheSisb** [comment, marked as answer]
Hi @jweintraub,

Brand owns Bynder and these assets, so I checked in with them on the matter. Their response was:
> you’ll want to avoid using Bynder as asset hosting, both due to it not being built for that purpose, as well as us not wanting to expose internal URLs in customer ui
> So - please copy and host as needed in your repos

Hope this helps clarify the usage!

**Reply by jweintraub** [reply]
Who did you speak with on the brand side? The direction I offered came from brand as well (Sang Seo). It sounds like there is some different opinions on that side. 

We should consider a way to host assets to use in pages in a scalable way. If not bynder, do you have another suggestion?

**Reply by TheSisb** [reply]
Hi @jweintraub,

I started a thread in the #mkg-brand channel on Slack. Andres Krogh responded.  

> We should consider a way to host assets to use in pages in a scalable way. If not bynder, do you have another suggestion?

I'd support that, however it is brand's call to make.

**Reply by jweintraub** [reply]
I will chime in on that thread. Thanks!
